backend/security/src/main/kotlin/io/tolgee/security/ratelimit/RateLimited.kt

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (C) 2023 Tolgee s.r.o. and contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.tolgee.security.ratelimit
+
+/**
+ * Enables rate limiting on a specific route in addition to the global rate limit.
+ * Has no effect when rate limiting is disabled in server configuration.
+ */
+@Target(AnnotationTarget.FUNCTION, AnnotationTarget.ANNOTATION_CLASS)
+annotation class RateLimited(
+  /**
+   * Maximum amount of requests that can be made during the window interval.
+   */
+  val limit: Int,
+  /**
+   * Size of the window to consider in milliseconds. Defaults to a second.
+   */
+  val windowSize: Long = 1_000,
```

Comment:
Can you please name in refillDurationInMs, so it's clear it's duration in milliseconds?